skonfig/cdist/arguments.py

```diff
@@ -9,6 +9,12 @@ def get(skonfig_arguments):
     if skonfig_arguments.manifest:
         cdist_argv.append("-i")
         cdist_argv.append(skonfig_arguments.manifest)
+    # no -j = single job
+    # only -j = defaults to cpu count
+    # -j n = n jobs
+    cdist_argv.append("-j")
```

Comment:
This always adds `-j` to `cdist_argv` essentially enabling parallelisation by default (unlike the comment above).

Reply:
Yes. My idea was that, by default, we always want this.

Reply:
Ah, my bad. Now everything makes sense.